src/main/java/CommandSet/FindCommand.java

```diff
@@ -0,0 +1,36 @@
+package CommandSet;
+
+import Helper.TaskList;
+import Helper.Ui;
+
+/**
+ * <h1>FindCommand</h1>
+ * <p>
+ * FindCommands checks if any of the tasks in the list of tasks match
+ * the user input word.
+ * </p>
+ *
+ * @author Saravanan Anuja Harish
+ */
+public class FindCommand extends Command {
+
+    // stores the command for find.
+    private static final String FIND = "find";
+
+    /**
+     * checks for tasks containing the word given by user.
+     *
+     * @param word the user input.
+     * @param taskList the list of user tasks.
+     */
+    public static void findTasksContaining(String word, TaskList taskList) {
+        word = word.substring(FIND.length()).trim();
```

Comment:
Perhaps, in order to prevent recycling variables, this `word` could be renamed to `processedWord`?